drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -469,7 +462,9 @@ function DragAndDropEditBlock(runtime, element, params) {
                             if (name.length > 0 || imageURL.length > 0) {
                                 var data = {
                                     displayName: name,
-                                    zone: $el.find('.zone-select').val(),
+                                    zones: $.map($el.find('.zone-checkbox:checked'), function(c){
```

Comment:
I think I might not be familiar enough with JS to follow this comment fully; I can definitely give `c` a better variable name, but I don't understand the construct you're suggesting - do you mean something like this?

```javascript
var checked_checkboxes = $el.find('.zone-checkbox:checked')
var data = {
    zones: $.map(checkboxes, function(checkbox){
        return checkbox.value();
    }
}
```